### PR TITLE
adding initial addons to tabs

### PR DESCRIPTION
**Pull request by mansona**
This is just splitting @MelSumner 's work from #558 into different PRs 👍

Fixes https://github.com/ember-learn/ember-website/issues/521

### CODE REVIEW

**Comment by MelSumner** [comment]
This one isn't quite ready for a PR, can you put this into draft mode for now?

**Comment by mansona** [comment]
@MelSumner sorry I can't change it back to draft 🤔 can you review it requesting changes so it can't be merged accidentally?

File: app/templates/components/addon-tabs.hbs [hunk 1]
```diff
@@ -5,21 +5,21 @@
       @tabId={{0}}
       @onClick={{action (mut this.currentTab) 0}}
     >
-      SASS
+      Manage State
```
Comment (Windvis) [inline app/templates/components/addon-tabs.hbs]: Maybe it's just me but when I read this title I never expected to find information about ember-concurrency. I immediately thought of things like Ember Data / Services / Redux / etc.

Maybe something like `Async Code` or `Async State` might be better here?

File: app/templates/components/addon-tabs.hbs [hunk 2]
```diff
@@ -28,53 +28,52 @@
...
   </div>
 
   {{!-- template-lint-disable block-indentation  --}}
   <AddonTabs::Panel
     @currentTab={{this.currentTab}}
     @tabId={{0}}
   >
-    <p>Ember CLI Sass wires up your app with Sass, in a single command.</p>
-
+    <p>Ember Concurrency is an Ember addon that makes state management a whole lot easier with useful primitives.</p>
```
Comment (Windvis) [inline app/templates/components/addon-tabs.hbs]: Same remark here. Maybe we can simply use the description from the [website](http://ember-concurrency.com/docs/introduction/)?

> ember-concurrency is an Ember Addon that makes it easy to write concise, robust, and beautiful asynchronous code

**Pull request merged**